statsmodels/regression/tests/test_lme.py

```diff
@@ -724,23 +767,27 @@ def test_summary(self):
         # Test that the summary correctly includes all variables.
         summ = self.res.summary()
         desired = ["const", "x1", "x2", "Group Var"]
-        actual = summ.tables[1].index.values # Second table is summary of params
+        actual = summ.tables[
+            1].index.values  # Second table is summary of params
```

Comment:
+1 for adding the extra space before the `#`, but does the line need to be split like this?  It's not a pattern most readers are used to.